_posts/2022-12-6-uma_outreachy.md

```diff
@@ -0,0 +1,61 @@
+---
+layout: post
+title: Outreachy Report - 2022 Improve MDAnalysis by implementing type hinting
+---
+
+## Motivation 
+
+The object-oriented Python library MDAnalysis analyzes trajectory data derived from molecular dynamics (MD) simulations in many popular formats.
+Python's dynamic nature enables us to develop with speed, flexibility, and ease of use, but if you are not careful, you can result in short-term expedience at the expense of long-term maintainability as a result of the dynamic nature of Python.
```

Comment:
but if you are not careful, you may trade short-term expedience for long-term lack of maintainability because of the dynamics nature of Python.